### PR TITLE
fix(i18n): add urdu localization

### PR DESCRIPTION
**Pull request by zairahira**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: i18n/locales/urdu/translations.json [hunk 1]
```diff
@@ -1,76 +1,80 @@
...
+    "load-more-articles": "مزید آرٹیکلز لوڈ کریں"
   },
   "search": {
     "label": "Search",
-    "accessible-name": "Submit your search query",
-    "placeholder": "Search 8,000+ tutorials",
-    "see-results": "See all results for {{ searchQuery }}",
-    "no-tutorials": "No tutorials found"
+    "accessible-name": "اپنی search query لکھیں ",
+    "placeholder": " 8,000+ tutorials",
```
Comment (scissorsneedfoodtoo) [inline i18n/locales/urdu/translations.json:10]: The number 8,000+ refers to the number of English tutorials / articles that have been published. Until a substantial number of articles have been published in Urdu, it's probably better to use a more generic term like "Search tutorials" for the placeholder string.

**Review by scissorsneedfoodtoo — COMMENTED** [review]
Hey @zairahira, thanks for opening up this new PR.

I tested locally and everything looks good. However, I do have a suggestion for the search bar placeholder. I'll leave that as a review comment.

Would you mind taking a look, and if you agree, could you update that string?

**Comment by raisedadead** [comment]
@scissorsneedfoodtoo do you know why these tests are failing?

File: i18n/locales/urdu/translations.json (hunk 1 above)
Comment (zairahira) [reply i18n/locales/urdu/translations.json:10]: > @scissorsneedfoodtoo do you know why these tests are failing?

According to logs, it seems like a connectivity timeout issue with localhost. But I can't see how it correlates with the changes in translations.

Logs:
`http://localhost:8080 timed out on retry 1228 of 41, elapsed 1230157ms, limit 1230000ms
Error: connect ECONNREFUSED [127](https://github.com/freeCodeCamp/news/runs/6117130638?check_suite_focus=true#step:6:127).0.0.1:8080`

**Comment by scissorsneedfoodtoo** [comment]
@raisedadead, I believe this is because it's coming from @zairahira's fork, not off a branch from the main repo like the other PRs. 

Does @zairahira need permission to push directly to upstream? Or should I do it on her behalf? Not sure what's easiest in this case.

**Comment by raisedadead** [comment]
Ah - I never noticed the part about the PR coming from a fork. I forgot that this particular project needs us to work on the upstream directly.

**Comment by raisedadead** [comment]
I have recreated this from a source branch on this repo. @zairahira you can delete your fork.

**Comment by scissorsneedfoodtoo** [comment]
Awesome, thanks @raisedadead. I'll work with @zairahira to update that branch and get these updates in.

**Pull request merged**